src/module/macros.ts

```diff
@@ -1251,14 +1226,14 @@ async function rollOverchargeMacro(actor: Actor, data: LancerOverchargeMacroData
 
 export async function prepareChargeMacro(a: string) {
   // Determine which Actor to speak as
-  let mech: LancerActor<EntryType.MECH> | null = getMacroSpeaker(a);
+  let mech = getMacroSpeaker(a);
   if (!mech) return;
   const ent = mech.data.data.derived.mm;
+  // TODO: Is this only supposed to run for NPCs?
```

Comment:
I don't believe PCs can have recharging systems, right?

Reply:
Is that what this is for? If so I'll just typeguard it to `this.is_npc()`